packages/theming/theming-utils/src/getCurrentAppearance.ts

```diff
@@ -2,5 +2,9 @@ import { Appearance } from 'react-native';
 import { AppearanceOptions, ThemeOptions } from '@fluentui-react-native/theme-types';
 
 export function getCurrentAppearance(appearance: ThemeOptions['appearance'], fallback: AppearanceOptions): AppearanceOptions {
+  if (appearance === undefined) {
```

Comment:
nit: I think `if (!!appearance)` does the same thing as this?

Reply:
I think `if (!!appearance)` only runs if appearance is a non-falsy (since !appearance is true if appearance is falsy and false if truthy, and then !!appearance would be false if falsy and true if truthy)

1. I personally prefer having bail outs earlier in functions and then running the rest of the logic instead of wrapping the logic in an if statement, I find it cleaner since it's less nesting and easier to read
2. The === undefined just lets me be more specific that I'm trying to catch undefined specifically instead of all falsy values. I don't mind changing it to `if (!appearance)`, but I like being more precise.

Reply:
Thanks for the explanation, yeah that makes sense!